runtime/integration-tests/src/xcm_transfers.rs

```diff
@@ -202,52 +204,54 @@ fn transfer_ksm_from_sibling() {
                         Parachain(PARA_ID_DEVELOPMENT),
                         Junction::AccountId32 {
                             network: Some(NetworkId::Kusama),
-                            id: SiblingKusamaReceiver::get().into(),
+                            id: SiblingReceiver::get().into(),
                         }
                     )
                 )
                 .into()
             ),
-            xcm_emulator::Unlimited
+            WeightLimit::Unlimited
         ));
     });
 
-    #[cfg(not(feature = "runtime-benchmarks"))]
-    Development::execute_with(|| {
-        let ksm_balance = OrmlTokens::free_balance(CurrencyId::KSM, &SiblingKusamaReceiver::get());
-        assert!(ksm_balance > 0);
-    });
+    // TODO: Fix this test
```

Comment:
whats up with this?

Reply:
@vstam1 out of curiosity why is the test failing?

Reply:
I fixed the test now. Looked like the relay chain wasn't processing the messages if `Kusama::execute_with` wasn't called.